jsonmerge_utils.py

```diff
@@ -5,17 +5,38 @@
 @author: GRENTOR
 """
 import os
+import argparse
 import logging
 #from tqdm import tqdm
 from merge_files import Merge
+
+parser = argparse.ArgumentParser(description="link-check is a broken link identifier")
+parser.add_argument('-ip',
+                    help="Input prefix",
+                    required=True)
+parser.add_argument('-op',
+                    help="Output prefix",
+                    required=True)
+parser.add_argument('-maxFileSize',
+                    help="The maximum file size (in bytes) that each merged file should have",
+                    required=True)
+parser.add_argument('-filepath',
+                    help="Path to directory where all json files are stored",
+                    required=True)
+parser.add_argument('-log_level',

```

Comment:
`dest = "log_level`

also add the **choices** and metavar parameters here as in:
`metavar = "{INFO','DEBUG','ERROR'}"`
`choices =  ['INFO', 'DEBUG', 'ERROR']`

metavar helps user to know he has only these choices